QATCH/ui/mainWindow_ui.py

```
@@ -590,17 +550,12 @@ def setRunMode(self, obj):
                     'Please "Analyze" to save or "Close" to lose your changes before switching modes.'
                 )
             else:
-                Log.e(
-                    "Please save your unsaved changes in VisQ.AI(tm) before switching modes."
-                )
+                Log.e("Please save your unsaved changes in VisQ.AI(tm) before switching modes.")
         if obj == None:
```

Ruff (0.15.9)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)